src/game/models/mod.rs

```diff
@@ -2,157 +2,17 @@
 
 use std::f64;
 use std::f64::consts::PI;
-use std::ops::{Add, AddAssign, Rem, RemAssign, Sub, SubAssign, Div, DivAssign};
 
 use opengl_graphics::GlGraphics;
-use piston_window::{Context, UpdateArgs, Size};
-use rand;
+use piston_window::{Context, UpdateArgs};
 
 pub mod player;
```

Comment:
Yeah let's just do it here, but you're right that this was an oversight from (probably me) from before.